2023-02-24 13:22:54,414 INFO    MainThread:191783 [wandb_setup.py:_flush():69] setting env: {}
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_setup.py:_flush():69] setting login settings: {}
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_init.py:_log_setup():334] Logging user logs to /home/renfei/Documents/se3gcnn/wandb/run-20230224_132254-10nug8nz/logs/debug.log
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_init.py:_log_setup():335] Logging internal logs to /home/renfei/Documents/se3gcnn/wandb/run-20230224_132254-10nug8nz/logs/debug-internal.log
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_init.py:init():367] calling init triggers
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2023-02-24 13:22:54,415 INFO    MainThread:191783 [wandb_init.py:init():416] starting backend
2023-02-24 13:22:54,416 INFO    MainThread:191783 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-02-24 13:22:54,423 INFO    MainThread:191783 [backend.py:ensure_launched():123] starting backend process...
2023-02-24 13:22:54,519 INFO    MainThread:191783 [backend.py:ensure_launched():128] started backend process with pid: 191821
2023-02-24 13:22:54,520 INFO    MainThread:191783 [wandb_init.py:init():421] backend started and connected
2023-02-24 13:22:54,521 INFO    MainThread:191783 [wandb_init.py:init():461] updated telemetry
2023-02-24 13:22:54,522 INFO    MainThread:191783 [wandb_init.py:init():480] communicating current version
2023-02-24 13:22:55,272 INFO    MainThread:191783 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.13.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-02-24 13:22:55,273 INFO    MainThread:191783 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2023-02-24 13:22:55,342 WARNING MainThread:191783 [wandb_init.py:init():763] interrupted
Traceback (most recent call last):
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 743, in init
    run = wi.init()
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 494, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 646, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 634, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 527, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
